contracts/AnalysisOptions.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace AMSMigrate.Contracts
 {
 
-    public record AnalysisOptions(string AccountName, DateTimeOffset? CreationTimeStart, DateTimeOffset? CreationTimeEnd, string? ResourceFilter, int BatchSize, bool IsStorageAcc =false);
+    public record AnalysisOptions(string AccountName, DateTimeOffset? CreationTimeStart, DateTimeOffset? CreationTimeEnd, string? ResourceFilter, int BatchSize, bool IsStorageAcc =false, string? Prefix = null);
```

Comment:
Prefix should be removed here,  ResourceFilter is already there and can be used as filter for Storage account.  ( filter just contains the prefix)